feature/automated-releases.feature

```diff
@@ -23,39 +27,56 @@ Feature: Automated releases
 
   Scenario: If a major release branch exists, the tool creates a major release from there
     Given following existing branches:
-      | name   |
-      | 1.0.x  |
-      | master |
+      | name    |
+      | 1.0.x   |
+      | 2.0.x   |
+      | master  |
+      | develop |
     And following open milestones:
       | name  |
       | 2.0.0 |
     When I close milestone "2.0.0"
     Then tag "2.0.0" should have been created on branch "master"
     And branch "2.0.x" should have been created from "master"
```

Comment:
This scenario confuses me:

- Why would the 2.0.0 tag be created from the `master` branch?
- Why would it create the `2.0.x` branch from `master`, considering the branch already exists?

Reply:
This is indeed an oversight: I will adjust 👍

Reply:
Adjusted: also, no branch created here: that's done by the `laminas:automatic-releases:switch-default-branch-to-next-minor` command instead.